src/bioregistry/data/contexts.json

```diff
@@ -0,0 +1,34 @@
+{
+  "obo": {
+    "custom_prefix_map": {
+      "DOI": "https://doi.org"
+    },
+    "description": "The OBO Context exists for the following reasons: ...",
```

Comment:
@matentzn this needs to be updated before we merge this PR

Reply:
1. Reconciling "database cross-references" used by OBO ontologies and centralising their validation.
2. Standardising and validating SSSOM mappings provided in the context of OBO ontologies.
3. Providing an incentive for the documentation of non-standard prefixes for OBO ontologies.

Reply:
Added in ff02f14